_posts/0000-01-02-pnet2log.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
So far, I've created a branch, crated a file and also a pull request. also was able to assigned myself, etc.